packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelForm.tsx

```diff
@@ -139,6 +156,14 @@ export default function ColumnModelForm(props: ColumnModelFormProps) {
     return {}
   }, [validationErrors])
 
+  const showError = !!errorsByField['maximumSize']
+
+  const helperText = errorsByField['maximumSize']
+    ? errorsByField['maximumSize']
+    : isSmallerThanRecommendedSize && recommendedSize
+    ? `Recommended size is at least ${recommendedSize}`
+    : ''
```

Comment:
Since this component shows multiple fields, suggest these renames:
```suggestion
  const showErrorOnMaxSizeField = !!errorsByField['maximumSize']

  const helperTextForMaxSizeField = errorsByField['maximumSize']
    ? errorsByField['maximumSize']
    : isSmallerThanRecommendedSize && recommendedSize
    ? `Recommended size is at least ${recommendedSize}`
    : ''
```